esp/logging/logginglib/logthread.cpp

```diff
@@ -92,6 +92,9 @@ CLogThread::CLogThread(IPropertyTree* _cfg , const char* _service, const char* _
 
         logFailSafe.setown(createFailSafeLogger(_service, _agentName, logsDir));
     }
+    time_t tNow;
+    time(&tNow);
+    localtime_r(&tNow, &m_startTime);
```

Comment:
@wangkx How does this relate to the other changes?  Was this just not initialized before?